contracts/src/v0.8/functions/tests/v1_0_0/FunctionsRouter.t.sol

```diff
@@ -1137,7 +1114,7 @@ contract FunctionsRouter_Fulfill is FunctionsClientRequestSetup {
       ) // TODO: build this programatically
     });
 
-    _reportAndStore(requestNumberKeys, results, errors, NOP_TRANSMITTER_ADDRESS_1, true, 1);
+    _reportAndStore(requestNumberKeys, results, errors, NOP_TRANSMITTER_ADDRESS_1, false, 1);
```

Comment:
@justinkaseman I need help to understand this `expectedToSucceed` argument in `_reportAndStore`.  I am not entirely sure why I had to adjust this value to get tests to pass after I made this minor change to how the `requestId` is generated.

I see that all it seems to do is [record the profit amounts](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/functions/tests/v1_0_0/Setup.t.sol#L443).  However it appears that this [consumes all the logs](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/functions/tests/v1_0_0/Setup.t.sol#L446) and it causes the [vm.expectEmit](https://github.com/smartcontractkit/chainlink/pull/10891/files#diff-5152fdbb2dac463e32e8e79ba3254a378b51e21693519523ff0c12a6d73b90b8L1126) to fail.

I am thinking this might be a simple semantic fix.  For example, instead of naming the argument `expectedToSucceed`, we could call it `recordCostAndProfit` or something.

Does this make sense or am I missing something?

Reply:
I figured this out.  I mistakenly left a `vm` cheat code which broke something. All tests now pass